MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -2000,50 +2053,32 @@
       "Id": "AreaSouthAccess",
       "RequiredItems": [],
       "ConditionalItems": [
-        [
-          "MaskDeku",
-          "ItemHookshot"
-        ],
-        [
-          "MaskDeku",
-          "OtherArrow"
-        ],
-        [
-          "MaskDeku",
-          "MaskZora"
-        ],
         [
           "ItemPictobox"
         ],
         [
           "Any Healing Potion"
         ],
         [
-          "Run Through Poisoned Water",
-          "Take Damage",
-          "MaskZora"
+          "Kill Big Octo",
+          "MaskDeku"
         ],
         [
+          "Kill Big Octo",
```

Comment:
How come this was added? Zora Mask already kills the Big Octo.

Reply:
The entry "Kill Big Octo" can probably be deleted if it includes Zora Mask. It was just done this way to make it abundantly clear in the editor what methods require killing the Big Octo or not.